content/unsupported-browser.html

```diff
@@ -4,15 +4,28 @@
 robots_tag: noindex
 ---
 
+<script defer src="/assets/js/unsupported-browser.js"></script>
+
 <section id="intro" class="intro-heading">
   <h1 class="mb-3">Unsupported Browser</h1>
   <p class="mb-1">
     Scratch Addons doesn't support the browser you're using.
   </p>
 </section>
 
-<section class="container">
-  <p>Use updated versions of Chrome, Firefox, or other browsers based on them to get started:</p>
+<div class="container">
+
+<section class="mb-5">

```

Comment:
What's with this large bottom margin?

Reply:
Looks better. See screenshot in OP

Reply:
I saw that earlier. I think it's an arbitrary space that doesn't help with anything but a waste of space, let alone "looked better".

Reply:
While the margin clearly separates the sections, it feels a little inconsistent and unnatural.

Reply:
Couldn't say that any better. I would say to just use `<hr>`, or something,